static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeDetector.kt

```diff
@@ -72,8 +72,12 @@ class AutoDisposeDetector: Detector(), SourceCodeScanner {
     context.project.propertyFiles.find { it.name == PROPERTY_FILE }?.apply {
       val content = StringReader(context.client.readFile(this).toString())
       props.load(content)
-      val scopes = props.getProperty(CUSTOM_SCOPE_KEY).split(",").map { it.trim() }
-      defaultScopes.addAll(scopes)
+      props.getProperty(CUSTOM_SCOPE_KEY)?.let { scopeProperty ->
```

Comment:
can we add a test for this?

Reply:
3ed7031e5da8c95f6723e3d2707d7e30291f7400